roles/devscripts/tasks/130_prep_host.yml

```diff
@@ -0,0 +1,66 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Ensure the required packages for executing dev-scripts exists.
+  tags:
+    - bootstrap
+    - packages
+  become: true
+  ansible.builtin.package:
```

Comment:
note that there are some packages coming from EPEL. In the reproducer, I could get around it with this:
https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/reproducer/tasks/devscripts_layout.yml#L2-L11